jobs/bump-jenkins-plugins.Jenkinsfile

```diff
@@ -0,0 +1,140 @@
+repo = "coreos/fedora-coreos-pipeline"
+fork_repo = "coreosbot-releng/fedora-coreos-pipeline"
+botCreds = "github-coreosbot-releng-token-username-password"
+pr_branch = "JenkinsPluginsUpdate"
+
+/* Function to extract the plugin version from the plugin URL */
+def getVersionFromPluginUrl(pluginUrl) {
+    /* example url : https://updates.jenkins.io/download/plugins/${pluginName}/latest/${pluginName}.hpi */
+    def parts = pluginUrl.split("/")
+    def pluginVersion
+    if (parts.size() >= 4) {
+        def groupId = parts[-3]
+        pluginVersion = parts[-2]
+    } else {
+        error("Unable to extract plugin version from the URL.")
+    }
+    return pluginVersion
+}
+
+node {
+    checkout scm: [
+        $class: 'GitSCM',
+        branches: [[name: "${pr_branch}"]],
+        userRemoteConfigs: [[url: "https://github.com/${fork_repo}.git"]],
+        extensions: [[$class: 'WipeWorkspace']]
+    ]
+    pipeutils = load("utils.groovy")
+
+    properties([
+        pipelineTriggers([
+            /* Schedule to check once a month */
+            pollSCM('H H 1 * *')
```

Comment:
this feels wrong. it feels like it should be `cron(` instead here. What source code are we polling?

Reply:
Yes, it should be cron here. I think I had pasted the syntax here. I will change it.